src/sdrf_pipelines/ols/ols.py

```
@@ -762,6 +762,39 @@ def search(self, term: str, ontology: str | None = None, exact=True, use_ols_cac
+        #   - None -> cannot be verified (cache-only: the parquet cache is label-indexed
+        #             with no synonyms/accession index; or a live lookup error) -> caller warns
+        if use_ols_cache_only:
+            return None
+        try:
+            docs = self.ols_search(
+                accession,
+                query_fields="obo_id",
+                field_list=["label", "obo_id", "synonym"],
+                exact=True,
+            )
+        except Exception as e:  # network / OLS error → cannot verify
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)